vagrant/ansible/roles/sit.cephfs/tasks/new_volume.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Create Ceph subvolume
+  command: >
+    /root/cephadm shell --
+      ceph fs subvolume create sit_fs {{ volume.name }}
+        --mode {{ volume.mode | default('0755') }}
+  run_once: true
+
+- name: Get subvolume path
+  command: >
+    /root/cephadm shell --
+      ceph fs subvolume getpath sit_fs {{ volume.name }}
+  run_once: true
+  register: ceph_subvol
+
+- name: Mount Ceph subvolume
+  mount:
+    path: "{{ volume.mount }}"
+    fstype: ceph
+    opts: "conf=/etc/ceph/sit.ceph.conf,name=sit"
```

Comment:
What does the `name=sit` option indicate?

Reply:
> What does the `name=sit` option indicate?

It's the name of the client to use to connect to the volume. It's defined in *vagrant/ansible/roles/sit.cephfs/tasks/main.yml*, in the authentication command as `client.sit`.